core/domain/activity_jobs_one_off.py

```diff
@@ -55,3 +56,39 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         pass
+
+
+class ReplaceAdminIdOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that replaces the usage of 'Admin' in ExplorationCommitLogEntryModel
+    and ExplorationRightsSnapshotMetadataModel.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationRightsSnapshotMetadataModel,
+                exp_models.ExplorationCommitLogEntryModel]
+
+    @staticmethod
+    def map(model):
+        if isinstance(model, exp_models.ExplorationRightsSnapshotMetadataModel):
+            if model.committer_id == 'Admin':
+                model.committer_id = feconf.SYSTEM_COMMITTER_ID
+                model.put(update_last_updated_time=False)
+                yield ('SUCCESS-DELETED-SNAPSHOT', model.id)
+            else:
+                yield ('SUCCESS-KEPT-SNAPSHOT', model.id)
+        else:
+            if model.user_id == 'Admin':
+                model.user_id = feconf.SYSTEM_COMMITTER_ID
+                model.put(update_last_updated_time=False)
+                yield ('SUCCESS-DELETED-COMMIT', model.id)
+            else:
+                yield ('SUCCESS-KEPT-COMMIT', model.id)
+
+    @staticmethod
+    def reduce(key, values):
+        """Implements the reduce function for this job."""
+        if key.startswith('SUCCESS-KEPT'):
+            yield (key, len(values))
+        else:
+            yield (key, values)
```

Comment:
I think it would be useful to have the length of these values to for easy verification after running the job (all models should be either deleted/kept).

Reply:
From the recent verification job we run on backup, this will be around 30 models and I want tho check that the ids that failed on the verification and the ids that are returned here are the same.